zips/zip-0233.md

```diff
@@ -19,224 +20,146 @@ The key words "MUST", "SHOULD", "SHOULD NOT", "MAY", "RECOMMENDED", "OPTIONAL",
-formula:
-
-$\mathsf{ZsfBalanceAfter}(\mathsf{height}) = \mathsf{MAX\_MONEY} + \sum_{h = 0}^{\mathsf{height}} (\mathsf{ZsfDeposit}(h) + \mathsf{Unclaimed}(h) - \mathsf{BlockSubsidy}(h))$
+The modifications required are:
 
-where $\mathsf{Unclaimed}(\mathsf{height})$ is the portion of the block subsidy
-and miner fees that are unclaimed for the block at the given height. 
+1. The addition of a transaction field representing an amount to burn for a
+   given transaction.
+2. The inclusion of the burn amount in the calculated total output value for a
+   given transaction.
+3. Consensus rules to ensure the burned amount is no longer available for
+   circulation on the network.
 
-The block at height $\mathsf{height}$ commits to
-$\mathsf{ZsfBalanceAfter}(\mathsf{height})$ as part of a new block commitment
-in the block header, at the end of the `hashBlockCommitments` chain in
-[ZIP-244](https://zips.z.cash/zip-0244#block-header-changes).
+## Transaction Format
 
-TODO ZIP editors: consider introducing a chain state commitment hash tree.
-(When we get closer to the network upgrade, we will have a better idea what
-commitments that network upgrade needs.)
+The following field is added to the v7 transaction format [3]:
 
-## Deposits into the Sustainability Fund
++-------+----------------+------------+------------------------------------------------------+
+| Bytes | Name           | Data Type  | Description                                          |
++=======+================+============+======================================================+
+|   8   | ``burnAmount`` | ``uint64`` | The amount of input value to be burned, in zatoshis. |
++-------+----------------+------------+------------------------------------------------------+
 
-Sustainability fund deposits can be made via the new `zsfDeposit` field. This
-can be done by:
-- ZEC fund holders, in non-coinbase transactions;
-- Zcash miners, in coinbase transactions.
+**Note:** Older transaction versions can continue to be supported after a
+network upgrade, but burning is not possible for these transactions. For
+example, NU5 supports both v4 and v5 transaction formats, for both coinbase and
+non-coinbase transactions.
 
-In transaction versions without this new field:
-- unclaimed miner fees and rewards in **coinbase transactions** are re-defined
-  as deposits into the sustainability fund, and
-- there are no sustainability fund deposits from non-coinbase transactions.
+## Consensus Rules
 
-Note: older transaction versions can continue to be supported after a network
-upgrade. For example, NU5 supports both v4 and v5 transaction formats, for both
-coinbase and non-coinbase transactions.
+The burned value must now be considered when calculating the total output value
+of a transaction. It should be treated similarly to a transparent output,
+except that there is no way for this value to be used as an input in a future
+transaction.
```

Comment:
Out interpretation of this (from which we will specify the actual consensus rules that replace this text during ZIP editing) is:

The burned value subtracts from the remaining value in the "transparent transaction value pool" as described in section 3.4: Transactions and Treestates and does not result in an output being produced in any chain value pool.

Reply:
This looks correct to me, yes. To clarify the intention: it should be possible to burn funds that originate from shielded inputs too. I _think_ this is included in the definition of "transparent transaction value pool"?

Reply:
That's correct, the "transparent transaction value pool" is _all_ value that is visible to chain observers; this includes the various valueBalance* fields and of course the explicitly transparent value.